<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>StreamPlay - Demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#topFrame {
border: none;
background: qradialgradient(cx:0.5, cy:0.2, radius: 1, fx:0.5, fy:0.1, 
stop:0 rgb(70, 70, 70), stop:1 rgb(0, 0, 0))
}
#botFrame {
border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.0 rgb(235, 235, 235), stop: 0.4 rgb(210, 210, 210),
stop: 0.5 rgb(210, 210, 210), stop: 0.74 rgb(195, 195, 195),
stop: 0.75 rgb(195, 195, 195), stop: 0.98 rgb(180, 180, 180));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblAction">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="StreamPlay.qrc">:/MainWindow/images/drop.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStatus">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>6</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #858585;
font-size:18pt;
font-weight:50;</string>
         </property>
         <property name="text">
          <string>Status Text</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSubStatus">
         <property name="styleSheet">
          <string notr="true">color: #858585;</string>
         </property>
         <property name="text">
          <string>Sub Status Text</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="botFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton { border: 0px; }
::menu-arrow { image: none; }
::menu-indicator { image:none; }
QMenu {
padding: 5px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="StreamPlay.qrc">
           <normaloff>:/MainWindow/images/airplay-off.png</normaloff>:/MainWindow/images/airplay-off.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>26</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionClear"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionOpen_File"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Subtitles"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuStreamPlay">
    <property name="title">
     <string>StreamPlay</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionStreamPlay_Help"/>
    <addaction name="actionReport_A_Problem"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <addaction name="menuStreamPlay"/>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About StreamPlay</string>
   </property>
  </action>
  <action name="actionLoad_Subtitles">
   <property name="text">
    <string>Load Subtitles...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionStreamPlay_Help">
   <property name="text">
    <string>Report an Issue...</string>
   </property>
  </action>
  <action name="actionReport_A_Problem">
   <property name="text">
    <string>StreamPlay Help</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Recent</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates...</string>
   </property>
  </action>
  <action name="actionCheck_For_Updates">
   <property name="text">
    <string>Check For Updates...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StreamPlay.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuOpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuClearRecentFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_Subtitles</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuLoadSubtitles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCheck_for_Updates</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuCheckForUpdates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPreferences</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReport_A_Problem</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuReportAProblem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStreamPlay_Help</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuOpenFile()</slot>
  <slot>menuClearRecentFiles()</slot>
  <slot>menuLoadSubtitles()</slot>
  <slot>menuCheckForUpdates()</slot>
  <slot>menuPreferences()</slot>
  <slot>menuAbout()</slot>
  <slot>menuHelp()</slot>
  <slot>menuReportAProblem()</slot>
 </slots>
</ui>
